0-ad-history-mod-package/simulation/templates/units/mace/champion_infantry_crossbowman.xml

```diff
@@ -5,6 +5,12 @@
     <Lang>greek</Lang>
     <SpecificName>Gastraphetophoros</SpecificName>
     <Icon>units/mace/champion_infantry_crossbowman.png</Icon>
+    <History>
+        The gastraphetes was a type of Greek crossbow, developed during the 5th or 4th century BC. Its name, deriving from its unique design, directly translates to “belly shooter” and, in a broader context, to “belly bow” which is the term used today.
+        The bow itself and the claw mechanism grabbing the bowstring were attached on opposite ends of two different beams of similar length. One of those beams (the one with the claw) rested inside a groove directly on the other beam, allowing both to slide back and forth along the other. With this particular arrangement, the bow and the claw mechanism could easily be brought together by pulling the top beam and/or pushing the lower beam. However, instead of then pulling back the bowstring like most crossbows do, the gastraphetes actually pushed the bow away from its string to create the tension: The upper beam, now protruding from the lower beam, was placed at a 45-degree angle forward on the ground. A big concave piece of wood attached to the lower beam's rear was placed to the shooter's belly. By leaning forward and pushing with his hands, pressing the crossbow against the ground, the shooter moved the beams together, therefore the bow away from its string, bending it, and therefore drawing the crossbow.
```

Comment:
```suggestion
        The bow itself and the claw mechanism grabbing the bowstring were attached on opposite ends of two different beams of similar length. One of those beams (the one with the claw) rested inside a groove directly on the other beam, allowing both to slide back and forth. With this particular arrangement, the bow and the claw mechanism could easily be brought together by pulling the top beam and/or pushing the lower beam. However, instead of then pulling back the bowstring like most crossbows do, the gastraphetes actually pushed the bow away from its string to create the tension: The upper beam, now protruding from the lower beam, was placed at a 45-degree angle forward on the ground. A big concave piece of wood attached to the lower beam's rear was placed to the shooter's belly. By leaning forward and pushing with his hands, pressing the crossbow against the ground, the shooter moved the beams together, therefore the bow away from its string, bending it, and therefore drawing the crossbow.
```